packages/launcher/__snapshots__/darwin_spec.ts.js

```diff
@@ -70,10 +50,47 @@ exports['darwin browser detection detects browsers as expected 1'] = [
     'findAppParams': {
       'appName': 'Google Chrome Canary.app',
       'executable': 'Contents/MacOS/Google Chrome Canary',
-      'appId': 'com.google.Chrome.canary',
+      'bundleId': 'com.google.Chrome.canary',
       'versionProperty': 'KSVersion',
     },
   },
+  {
+    'name': 'chrome-for-testing',
+    'family': 'chromium',
+    'channel': 'stable',
+    'displayName': 'Chrome Test',
+    'versionRegex': {},
+    'binary': 'chrome',
+    'minSupportedVersion': 64,
```

Comment:
113 was the first release of Chrome for Testing

See https://googlechromelabs.github.io/chrome-for-testing/files

Reply:
I decided to leave as `64` since it's a [shared value](https://github.com/cypress-io/cypress/blob/74cf21643eac4e40e6d1ceb0058e9d841d374bee/packages/launcher/lib/known-browsers.ts#L4) between all of the Chrome browsers and since it's prior to `113` it shouldn't have an impact.